tests/Unit/NumericalAlgorithms/Spectral/Test_ComplexDataView.cpp

```diff
@@ -215,16 +210,12 @@ SPECTRE_TEST_CASE("Unit.NumericalAlgorithms.Spectral.ComplexDataView",
   ASSERTION_TEST();
 #ifdef SPECTRE_DEBUG
   MAKE_GENERATOR(gen);
-  UniformCustomDistribution<double> dist{-100.0, 100.0};
   UniformCustomDistribution<size_t> sdist{5, 50};
   size_t overall_size = sdist(gen);
```

Comment:
Here and in the previous you can `const` the const things (mostly to push good habits :) )